### ISSUE TITLE
Krun will crash (and not bring the network back up) if the ssh identity file is absent.

### ISSUE THREAD

**Issue by vext01** [softdevteam/warmup_experiment#398]
```
2017-04-26 14:01:26: WARNING] SIMULATED: `time.sleep(60)` (--quick)
[2017-04-26 14:01:26: DEBUG] Taking fresh initial temperature readings
[2017-04-26 14:01:26: DEBUG] Set start temperatures: {'coretemp:temp4_input': 27.0, 'coretemp:temp5_input': 29.0, 'coretemp:temp2_input': 32.0, 'coretemp:temp3_input': 30.0, 'coretemp:temp1_input': 32.0}
[2017-04-26 14:01:26: DEBUG] Reading status cookie from warmup.manifest
[2017-04-26 14:01:26: DEBUG] Writing results out to: warmup_results.json.bz2
[2017-04-26 14:01:26: DEBUG] execute shell cmd: sudo systemctl start cron || true
[2017-04-26 14:01:26: DEBUG] execute shell cmd: sudo systemctl start atd || true
[2017-04-26 14:01:26: DEBUG] execute shell cmd: sudo systemctl start postfix || true
[2017-04-26 14:01:26: DEBUG] execute shell cmd: tar czf - ${KRUN_RESULTS_FILE} ${KRUN_LOG_FILE} ${KRUN_MANIFEST_FILE} | ssh -o 'BatchMode yes' -i id_rsa vext01@bencher2.soft-dev.org 'cat > research/krun_results//bencher7.tgz' || ( sleep 2; tar czf - ${KRUN_RESULTS_FILE} ${KRUN_LOG_FILE} ${KRUN_MANIFEST_FILE} | ssh -o 'BatchMode yes' -i id_rsa vext01@bencher2.soft-dev.org 'cat > research/krun_results//bencher7.tgz' ) || true 
[2017-04-26 14:01:28: DEBUG] execute shell cmd: ssh -o 'BatchMode yes' -i id_rsa vext01@bencher2.soft-dev.org 'test ! -e research/krun_results//bencher7.stop'
[2017-04-26 14:01:28: ERROR] Command failed: 'ssh -o 'BatchMode yes' -i id_rsa vext01@bencher2.soft-dev.org 'test ! -e research/krun_results//bencher7.stop''
stdout:

stderr:
Warning: Identity file id_rsa not accessible: No such file or directory.
Host key verification failed.


[2017-04-26 14:01:28: DEBUG] Fatal Krun error: Command failed: 'ssh -o 'BatchMode yes' -i id_rsa vext01@bencher2.soft-dev.org 'test ! -e research/krun_results//bencher7.stop''
stdout:

stderr:
Warning: Identity file id_rsa not accessible: No such file or directory.
Host key verification failed.


  File "./krun/krun.py", line 194, in main
    inner_main(mailer, on_first_invocation, config, args)
  File "./krun/krun.py", line 300, in inner_main
    util.run_shell_cmd_list(config.POST_EXECUTION_CMDS,)
  File "/home/vext01/warmup_experiment/krun/krun/util.py", line 114, in run_shell_cmd_list
    _, _, rv = run_shell_cmd(cmd, extra_env=extra_env)
  File "/home/vext01/warmup_experiment/krun/krun/util.py", line 93, in run_shell_cmd
    fatal(msg)
  File "/home/vext01/warmup_experiment/krun/krun/util.py", line 52, in fatal
    raise FatalKrunError(msg)

Traceback (most recent call last):
  File "./krun/krun.py", line 358, in <module>
    main(parser)
  File "./krun/krun.py", line 203, in main
    raise exn
krun.util.FatalKrunError: Command failed: 'ssh -o 'BatchMode yes' -i id_rsa vext01@bencher2.soft-dev.org 'test ! -e research/krun_results//bencher7.stop''
stdout:

stderr:
Warning: Identity file id_rsa not accessible: No such file or directory.
Host key verification failed.
```

We should allow the stop cookie check command to fail. I'll also add an early check that the identity file exists.

**Comment by vext01** [comment]
Hrm, actually it's unclear why the network didn't come up:

I've put a echo in place of the real net up command.  It should have occurred before the crash.

```
2017-04-26 14:21:48: DEBUG] Reading status cookie from warmup.manifest
[2017-04-26 14:21:48: DEBUG] Writing results out to: warmup_results.json.bz2
[2017-04-26 14:21:48: DEBUG] execute shell cmd: echo NETUP <<<<<<<<<<<<<<
[2017-04-26 14:21:48: DEBUG] execute shell cmd: sudo systemctl start cron || true
[2017-04-26 14:21:48: DEBUG] execute shell cmd: sudo systemctl start atd || true
[2017-04-26 14:21:48: DEBUG] execute shell cmd: sudo systemctl start postfix || true
[2017-04-26 14:21:49: DEBUG] execute shell cmd: tar czf - ${KRUN_RESULTS_FILE} ${KRUN_LOG_FILE} ${KRUN_MANIFEST_FILE} | ssh -o 'BatchMode yes' -i id_rsa vext01@bencher2.soft-dev.org 'cat > research/krun_results//bencher7.tgz' || ( sleep 2; tar czf - ${KRUN_RESULTS_FILE} ${KRUN_LOG_FILE} ${KRUN_MANIFEST_FILE} | ssh -o 'BatchMode yes' -i id_rsa vext01@bencher2.soft-dev.org 'cat > research/krun_results//bencher7.tgz' ) || true 
[2017-04-26 14:21:51: DEBUG] execute shell cmd: ssh -o 'BatchMode yes' -i id_rsa vext01@bencher2.soft-dev.org 'test ! -e research/krun_results//bencher7.stop'
[2017-04-26 14:21:51: ERROR] Command failed: 'ssh -o 'BatchMode yes' -i id_rsa vext01@bencher2.soft-dev.org 'test ! -e research/krun_results//bencher7.stop''
stdout:

stderr:
Warning: Identity file id_rsa not accessible: No such file or directory.
Permission denied (publickey,password).
...
```

**Comment by vext01** [comment]
Fixed by #399
